AbbreviationAsWordInName/Example1/list-of-projects.properties

```diff
@@ -39,19 +39,18 @@ RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
 Vavr|git|https://github.com/vavr-io/vavr|v0.9.0||
 
 # For special checks only
-
 # Few projects with excludes to decrease a number of checked files (useful for some checks which specify overly strong code style policy)
-checkstyle-with-excludes|git|https://github.com/checkstyle/checkstyle.git|master|**/checkstyle-with-excludes/src/test/**/*,**/checkstyle-with-excludes/src/it/resources/**/*,**/resources-noncompilable/**/*
-sevntu-checkstyle-with-excludes|git|https://github.com/sevntu-checkstyle/sevntu.checkstyle|master|**/sevntu-checkstyle-with-excludes/sevntu-checks/src/test/**/*
-spotbugs-with-excldues|git|https://github.com/spotbugs/spotbugs|3.1.2|**/spotbugs-with-excldues/eclipsePlugin-test/**/*,**/spotbugs-with-excldues/spotbugsTestCases/**/*,**/spotbugs-with-excldues/JSR305-testCases/**/*
-hibernate-orm-with-excludes|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|**/hibernate-orm-with-excludes/documentation/**/*,**/hibernate-orm-with-excludes/**/src/test/**/*
+#checkstyle-with-excludes|git|https://github.com/checkstyle/checkstyle.git|master|**/checkstyle-with-excludes/src/test/**/*,**/checkstyle-with-excludes/src/it/resources/**/*,**/resources-noncompilable/**/*
```

Comment:
please remove, not a comment.
no need to keep this noise.

Reply:
so, remove all after the for special checks for these folders and keep the javadoc project for summary javadoc?

Reply:
basically, remove everything starting from #For special checks only?

Reply:
just remove all commented out project line in this file, wedo not need keep them.
file should be easy to read.

Reply:
okay, i just to clarity. the original issue was about special checks, right? and, i put them under special checks. to my understanding there was no duplication. just the changing of heading and moved some components under special check. Now, do you want me to remove those under special checks?

Reply:
> please remove, not a comment. no need to keep this noise.

done

Reply:
>Now, do you want me to remove those under special checks?

if you can create RequiredThis configs, this project will be main project in list.
as majority of projects do not use `this` extensively, so throught a time we will collect such crazy projects about this and they will be in our set of pojects for this RequiredThis